src/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { getCommonProps, noOp } from '../../utils';
+import classnames from 'classnames';
+import * as Dialog from '@radix-ui/react-dialog';
+import CloseIcon from '../../assets/close.svg?react';
+import IconButton from '../IconButton/IconButton';
+
+// You'll need to change the HTMLDivElement to match the top-level element of your component
+export interface DrawerProps extends React.HTMLAttributes<HTMLDivElement> {
+  isOpen?: boolean;
```

Comment:
interface needs some comments for docs

Reply:
@davidicus can I add it in new Footer's PR? so we can merge this one and I can finish Footer today